packages/url_launcher/url_launcher_windows/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.1.4
+
+* Fixes an issue where the URL logged would not be escaped on failure.
```

Comment:
Shouldn't this say "would not be unescaped"?

Reply:
Done!